cyral/data_source_cyral_sidecar_bound_ports.go

```diff
@@ -182,13 +182,6 @@ func getBindingPorts(
 		bindingPorts = append(bindingPorts, primaryPort)
 	}
 
-	// MaxAllowedListeners is currently used to generate a repo binding port range
-	// between primaryPort and (primaryPort+MaxAllowedListeners-1). Thats why we
-	// add (MaxAllowedListeners-1) more ports to the repo binding ports.
-	for i := uint32(1); i < repoInfo.MaxAllowedListeners; i++ {
```

Comment:
This will be done in the changes to cyral_repository_binding, which can be found [here](https://github.com/cyralinc/terraform-provider-cyral/pull/328).